__tests__/nc-games.test.js

```diff
@@ -52,7 +52,8 @@ describe("/api/reviews/:review_id", () => {
                 review_body: 'Farmyard fun!',
                 category: 'euro game',
                 created_at: "2021-01-18T10:00:20.514Z",
-                votes: 1
+                votes: 1,
+                comment_count: "0"
```

Comment:
when you add a new feature you should create a new and separate test, not alter previous tests. If the original test failed with the changes it means it was a brittle test.

Reply:
I was a little unsure on this, so thanks for clarification. Going to refactor that test to satisfy only the previous condition (without breaking when the new property is introduced) and then create a new test to test for the new property.
Thanks!